packages/node_modules/@webex/internal-plugin-conversation/src/conversation.js

```diff
@@ -1115,30 +1130,65 @@ const Conversation = WebexPlugin.extend({
    * @private
    * @returns {Promise<Array<Conversation>>}
    */
-  _list(options) {
+  async _list(options) {
     options.qs = Object.assign({
       personRefresh: true,
       uuidEntryFormat: true,
       activitiesLimit: 0,
       participantsLimit: 0
     }, options.qs);
 
-    return this.request(options)
-      .then((res) => {
-        if (!res.body || !res.body.items || res.body.items.length === 0) {
-          return [];
-        }
+    const res = await this.request(options);
+
+    if (!res.body || !res.body.items || res.body.items.length === 0) {
+      return [];
+    }
+
+    let list = res.body.items.slice(0);
+
+    if (last(list).published < list[0].published) {
+      list.reverse();
+    }
+
+    if (res.body.additionalUrls) {
+      let limit = 0;
+
+      if (options.limit) {
+        limit = Math.floor(
+          (options.limit.value - list.length) / res.body.additionalUrls.length
+        );
+      }
+
+      if (!options.limit || limit !== 0) {
+        const results = await Promise.all(
+          res.body.additionalUrls.map((host) => {
+            const url = `${host}/${options.resource}`;
+            const newOptions = Object.assign({}, options, {uri: url, url});
```

Comment:
This works, but you can use the spread operator here.

Reply:
Turns out. This is faster than the spread operator. If you prefer the spread operator, I can definitely change it.

Reply:
Nope, not blocking. Just trying to get the codebase consistent where possible. 

That's interesting that it's faster. Good to know.